packages/react-input/Spec.md

```diff
@@ -1,63 +1,322 @@
+| value                        | `value?: string`                                      | `value?: string \| number`                                  | `value?: string`                        |
+| defaultValue                 | `defaultValue?: string`                               | `defaultValue?: string \| string[]`                         | `defaultValue?: string`                 |
+| onChange                     | `(ev, value) => void`                                 | `(ev, data: { ...props, value }) => void`                   | `(ev, data: { value: string }) => void` |
+| container className          | `className?: string`                                  |                                                             | top-level `className`                   |
+| input className              | `inputClassName?: string`                             | `input.className?: `                                        |                                         |
+| aria label                   | `ariaLabel?: string`                                  | `'aria-label'?: string`                                     | `'aria-label'?: string`                 |
+| clearable                    | n/a                                                   | `clearable?: boolean`                                       | TBD (deferred)                          |
+| theme                        | `theme?: ITheme`                                      |                                                             | handled by context                      |
+| style overrides              | `styles?: IStyleFunctionOrObject<...>`                | `styles?: ComponentSlotStyles<...>`                         | className, slot classNames              |
+| accessibility                | individual props                                      | `Accessibility<InputBehaviorProps>` (see below)             | individual props?                       |
+
+```ts
+// packages/fluentui/react-northstar/src/utils/htmlPropsUtils.tsx
+type SupportedIntrinsicInputProps = {
+  [K in HtmlInputProps]?: K extends keyof JSX.IntrinsicElements['input'] ? JSX.IntrinsicElements['input'][K] : any;
+};
+// HtmlInputProps: a subset of React and HTML native props
+
+// packages/fluentui/react-northstar/src/components/Box/Box.tsx
+interface BoxProps extends UIComponentProps<BoxProps>, ContentComponentProps, ChildrenComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility<never>;
+}
+
+// packages/fluentui/accessibility/src/behaviors/Input/inputBehavior.ts
+type InputBehaviorProps = { disabled?: boolean; required?: boolean; error?: boolean };
+```
+
+Native props following standard behavior in both libraries + converged:
+
+- `disabled?: boolean`
+- `readOnly?: boolean` -- VERIFY v0
+- `autoComplete?: string` -- VERIFY v0
```

Comment:
Should we mark these as TODOs so that they are easily searchable? Unless you are planning on making the verification before merging this PR, in which case it is fine as long as we remove these comments.